ui/src/app/base/actions.js

```diff
@@ -10,7 +12,7 @@ export const fetchAuthUser = () => {
     meta: {
       model: "user",
       method: "auth_user",
-      type: 0
+      type: MESSAGE_TYPES.REQUEST
```

Comment:
As discussed let's move this into the saga as they always need to be the same type.